WalletWasabi.Gui/Controls/WalletExplorer/SendControlView.xaml

```diff
@@ -87,7 +88,7 @@
               <StackPanel Orientation="Horizontal" VerticalAlignment="Center" Spacing="4">
                 <TextBlock Text="Confirmation Expected In:" VerticalAlignment="Center" />
                 <TextBlock Foreground="YellowGreen" Text="{Binding FeeTarget, Converter={StaticResource FeeTargetTimeConverter}}" Width="80" VerticalAlignment="Center" />
-                <Button Content="{Binding FeeText}" ToolTip.Tip="{Binding FeeToolTip}" Command="{Binding FeeRateCommand}" Padding="0" Background="{DynamicResource EditorBackgroundBrush}" BorderBrush="{DynamicResource EditorBackgroundBrush}" VerticalAlignment="Top" />
+                <Button DataContext="{Binding TransactionFeeInfo, Converter={StaticResource FeeDisplayFormatConverter}}" Content="{Binding FeeText}" ToolTip.Tip="{Binding FeeToolTip}" Command="{Binding RelativeSource={RelativeSource AncestorType=StackPanel}, Path=DataContext.FeeRateCommand}" Padding="0" Background="{DynamicResource EditorBackgroundBrush}" BorderBrush="{DynamicResource EditorBackgroundBrush}" VerticalAlignment="Top" />
```

Comment:
@jmacato can you take a look at this? Seems a bit complex to me. Can this be simplified?